src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
```

Comment:
@stephentoub when invoking RemoteInvoke in UAP we don't create a new process and instead we activate a WinRT background task to execute the code remotely. RemoteInvoke always return null and never return process handle. I don't think this test is applicable on UAP and we just need to exclude UAP without adding any test for it.